R/sfa_get_entities.R

```diff
@@ -6,7 +6,7 @@ sfa_get_entities <- function(
   api_key = getOption("sfa_api_key"),
   cache_dir = getOption("sfa_cache_dir")
 ) {
-  check_api_key(api_key)
+  #check_api_key(api_key)
```

Comment:
rather than uncommenting this, would be better maybe to modify check_api_key (remove char limit) and then to use the check_api_key function also in sfa_set_api_key

Reply:
I agree. This means:

- [x] Modify `check_api_key()` 
- [ ] Use `check_api_key()`  in `sfa_set_api_key()` as well
- [ ] Adapt the tests to the new API key patterns